odl/solvers/advanced/proximal_operators.py

```diff
@@ -208,17 +385,18 @@ def proximal_convexconjugate_l2(space, lam=1, g=None):
     Function for the proximal operator of the convex conjugate of the
     functional F where F is the l2-norm
 
-        F(x) =  lam 1/2 ||x - g||_2^2
+        F(x) =  lam ||x - g||_2
 
     with x and g elements in ``space``, scaling factor lam, and given data g.
 
-    The convex conjugate, F_cc, of F is given by
+    The convex conjugate, F^*, of F is given by
 
-        F_cc(y) = 1/lam (1/2 ||y/lam||_2^2 + <y/lam,g>)
+        F^*(y) = {0 if ||x-g|| < lam, infty else}
 
-    The proximal operator of F_cc is given by
+    The proximal operator of ``sigma * F^*`` where ``sigma`` is a step size
+    is given by
 
-        prox_sigma[F_cc](y) = (y - sigma g) / (1 + sigma/lam)
+        prox[sigma * F^*](y) = (y - sigma g) / (1 + sigma/lam)
```

Comment:
So thats where my confusion came from, this was obviously copy-paste. Thought you commented on the other place (where it should be correct).